ubr/utils.py

```diff
@@ -132,6 +133,52 @@ def pairwise(lst):
     iterator = iter(lst)
     return izip(iterator, iterator)
 
+def enumerated(lst):
+    return dict(zip(range(1, len(lst) + 1), lst))
+
+def isint(x):
```

Comment:
that assumes input is a string with a method `isdigit`. I could do `str(x).isdigit()` but it's really not worth it. the `isint` implementation is something I've been doing for years